setup.md

```diff
@@ -3,13 +3,42 @@ layout: page
 title: Setup
 ---
 
+> ## Data
+> Data for this lesson is from the [The SAFI Teaching Database](http://datacarpentry.org/socialsci-workshop/data/).
+>
+> We will use the files listed below for the data in this lesson. You can download the files by clicking on the links.
+>
+> - [SAFI_clean.csv](/data/SAFI_clean.csv)
+> - [SAFI_results.csv](/data/SAFI_results.csv)
+> - [SAFI\_full\_shortname.csv](/data/SAFI_full_shortname.csv)
+> - [SAFI.json](/data/SAFI.json)
```

Comment:
Hi @katrintirok resurrecting this old PR to ask you to add a link to the data file used in `08-Pandas` (SN7577.tab) to this list, then I will go ahead and merge this.

Reply:
I added the link to the SN7577 data file

Reply:
Thank you Katrin!